tests/core/filtering/test_contract_topic_filters.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 
-from hypothesis import (
+from web3.tools.hypothesis import (
```

Comment:
Same, import should come directly from the `hypothesis` module (not proxied through the `web3.tools.hypothesis` imports.